config/config.exs

```diff
@@ -215,6 +215,9 @@ config :bodyguard,
 
 config :flop, repo: Trento.Repo
 
+config :trento,
+  admin_username: "admin"
```

Comment:
nitpick: Maybe just to match with the env variable, could we name this `admin_user`?
As the env variable is `ADMIN_USER`

Reply:
done!